tests/Klasse.spec.ts

```diff
@@ -133,4 +133,52 @@ test.describe(`Testfälle für die Administration von Klassen: Umgebung: ${proce
       await expect(klasseCreationView.button_ZurueckErgebnisliste).toBeVisible();
     });
   });
+
+  test("Jede Klasse hat eine Dienststellennummer neben dem Klassennamen (mit Paginierung)", { tag: [LONG, SHORT, STAGE] }, async ({ page }) => {
+    const startseite = new StartPage(page);
+    const menue = new MenuPage(page);
+    const klasseManagementView = new KlasseManagementViewPage(page);
+
+    await test.step(`Klassenverwaltung öffnen und prüfen, dass jede Klasse eine Dienststellennummer hat`, async () => {
+        // Navigate to Klassenverwaltung
+        await startseite.card_item_schulportal_administration.click();
+        await menue.menueItem_AlleKlassenAnzeigen.click();
+
+        // Wait until the table is visible
+        await expect(klasseManagementView.text_h2_Klassenverwaltung).toHaveText("Klassenverwaltung");
+
+        // Define the selector for the 'Next' button
+        const nextPageButton = page.locator('.v-pagination__next button:not(.v-btn--disabled)');
```

Comment:
Please define locators in the page files. In this case FooterDataTable.page.ts. 
(playwright inspector helps to create locators: npx playwright codegen https://main.dev.spsh.dbildungsplattform.de/)
e.g. await page.getByLabel('Nächste Seite').click();

Reply:
Locators for last page are now defined in the main KlassenmanagementPage